kie-server-parent/kie-server-controller/kie-server-controller-api/src/main/java/org/kie/server/controller/api/commands/KieServerControllerDescriptorCommand.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.controller.api.commands;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElements;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import org.kie.server.api.model.KieServerCommand;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.controller.api.model.runtime.ServerInstanceKey;
+import org.kie.server.controller.api.model.spec.*;
+
+@XmlRootElement(name = "controller-descriptor-command")
+@XmlAccessorType(XmlAccessType.NONE)
+public class KieServerControllerDescriptorCommand implements KieServerCommand {
+
+    private static final long serialVersionUID = -1803374525440238418L;
+
+    @XmlElement(name = "service")
+    private String service;
+
+    @XmlElement(name = "method")
+    private String method;
+
+    @XmlElements({
+            @XmlElement(name = "container-spec-details", type = ContainerSpec.class),
+            @XmlElement(name = "server-template-details", type = ServerTemplate.class),
+            @XmlElement(name = "rule-config", type = RuleConfig.class),
+            @XmlElement(name = "process-config", type = ProcessConfig.class),
+            @XmlElement(name = "server-config", type = ServerConfig.class),
+            @XmlElement(name = "container-config", type = ContainerConfig.class),
+            @XmlElement(name = "container-spec-key", type = ContainerSpecKey.class),
+            @XmlElement(name = "server-instance-key", type = ServerInstanceKey.class),
+            @XmlElement(name = "server-instance-key", type = ReleaseId.class),
```

Comment:
I think, that here should be name "release-id". That's XmlRootElement name for class [ReleaseId](https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/ReleaseId.java)

Reply:
@jakubschwan thanks, fixed now.